migrations/2022-01-04-034553_add_hidden_column/up.sql

```diff
@@ -0,0 +1,20 @@
+alter table community add column "hidden" BOOLEAN DEFAULT FALSE;
+
+
+CREATE TABLE IF NOT EXISTS mod_hide_community
+(
+   id serial primary key,
+    community_id integer NOT NULL,
+    person_id integer NOT NULL,
+    when_ timestamp without time zone NOT NULL DEFAULT now(),
+    reason text,
+    CONSTRAINT mod_hide_community_id_fkey FOREIGN KEY (community_id)
+        REFERENCES community (id) MATCH SIMPLE
+        ON UPDATE CASCADE
+        ON DELETE CASCADE,
+    CONSTRAINT mod_hide_community_person_id_fkey FOREIGN KEY (person_id)
+        REFERENCES person (id) MATCH SIMPLE
+        ON UPDATE CASCADE
+        ON DELETE CASCADE
+)
```

Comment:
I've already reminded you to change this to be like the other examples, this is not how we are adding constraints. look at the `mod_transfer_community` example.

Reply:
Thank you, I was confused on which one to base it off of. I'll post it in the chat next time.